samples/browser/src/main/assets/extensions/helloworld/helloworld.js

```diff
@@ -0,0 +1 @@
+console.log("Hello World!");
```

Comment:
If we want something more visible then we could do something like `borderify.js` from the WebExtension samples:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Your_first_WebExtension#borderify.js

Reply:
Add a border to the helloworld extension for now! :)